core/network/space.go

```diff
@@ -89,3 +99,28 @@ func (s SpaceInfos) GetByName(name string) *SpaceInfo {
 	}
 	return nil
 }
+
+// ContainsID returns true if the collection contains a
+// space with the given ID.
+func (s SpaceInfos) ContainsID(id string) bool {
+	return s.GetByID(id) != nil
+}
+
+// ContainsName returns true if the collection contains a
+// space with the given name.
+func (s SpaceInfos) ContainsName(name string) bool {
+	return s.GetByName(name) != nil
+}
+
+// Difference returns a new SpaceInfos representing all the
+// values in the target that are not in the parameter. Value
+// matching is done by ID.
+func (s SpaceInfos) Difference(other SpaceInfos) SpaceInfos {
```

Comment:
I know this simulates the same method on sets, but I think names like `NotIn` or `Minus` better convey the behaviour. 

`Difference` says everything non-intersecting to me.

Reply:
done